dist/stat_data.py

```diff
@@ -551,7 +551,10 @@ def __init__(self, name, desc, flags=''):
     TxnStat('txn_prepare_active', 'prepared transactions currently active'),
     TxnStat('txn_prepare_commit', 'prepared transactions committed'),
     TxnStat('txn_prepare_rollback', 'prepared transactions rolled back'),
+    TxnStat('txn_prepared_updates_commit_count', 'Number of prepared updates committed'),
     TxnStat('txn_prepared_updates_count', 'Number of prepared updates'),
+    TxnStat('txn_prepared_updates_key_repeated_count', 'Number of prepared updates repeated on the same key'),
+    TxnStat('txn_prepared_updates_rollback_count', 'Number of prepared updates rollback-ed'),
```

Comment:
I would rather add the name "rollbacked" to the known word list.

Reply:
Fixed.